sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -177,10 +173,8 @@ export interface RecognizeLinkedEntitiesSuccessResult extends TextAnalyticsSucce
 // @public
 export interface SentenceSentiment {
     confidenceScores: SentimentConfidenceScores;
-    graphemeLength: number;
-    graphemeOffset: number;
     sentiment: SentenceSentimentLabel;
-    warnings?: string[];
+    text?: string;
```

Comment:
When is text omitted from this type?

Reply:
It's not marked required in the service swagger.

Reply:
👍